### ISSUE TITLE
Add version of ComfyUI to the saving workflow file

### ISSUE THREAD

**Issue by M4Mahdi** [comfyanonymous/ComfyUI#5200]
### Feature Idea

Why not add version of ComfyUI to the saving workflow file?

I found a community that users shared their workflows there and there was some shining workflows to use for my purposes. But some/many of nodes was red after loading them to the ComfyUI 0.2.2(latest version in this time). There were missing/undefined node types due to using the old versions of ComfyUI by workflow providers. I thought that if there was version of used ComfyUI in that workflow files(in the workflow file when i open it via notepad or when the ComfyUI shows me the missing popup message), i could download that version of ComfyUI from Github release page and load workflows there to use. This can be easier way than "editing that workflow to solve problems" for some users and situations.

Beside that, isn't it standard if the version of ComfyUI be in the workflow file?

**Comment by LukeG89** [comment]
You should request it in the frontend repo: https://github.com/Comfy-Org/ComfyUI_frontend

**Comment by ltdrdata** [comment]
That's not the ComfyUI version. From when ComfyUI was first released until now, it has consistently been 0.4.
While it seems more reasonable to view it as the workflow format version, it still feels like the version should have changed at least a few times by now. lol

**Comment by M4Mahdi** [comment]
> That's not the ComfyUI version. From when ComfyUI was first released until now, it has consistently been 0.4. While it seems more reasonable to view it as the workflow format version, it still feels like the version should have changed at least a few times by now. lol

You're right. At least, we can understand now why there are versions below 1.0 😃😌

So i think it is necessary to have a plan for this to solve the issue as soon as possible:
Versioning workflow format on each release of ComfyUI and put it in the saving workflow file.

Also, i think it is necessary to know the corresponding "ComfyUI version" for "workflow format version" that is in the workflow.json file. Because i, as a normal user, still don't know the ComfyUI version that is suitable for my old workflow file even if know the workflow format version. It(ComfyUI version) can save in the workflow.json file too or be in another easy accessible place for me as a normal user.
